docs/tips/gallery.md

```diff
@@ -77,3 +77,25 @@ show(p)
 ```
 
 ![history_sample](../imgs/history_sample.png)
+
+## Callback Scheduling Visualization
+Callback Scheduling Visualization can visualize the callback scheduling of targets such as a Node, Executor, and Callbackgroup.
+```python
+# traget: node 
+node = app.get_node('node_name') # get node object
+callback_sched(node)
+# traget: executor 
+executor = app.get_executor('executor_name') # get executor object
+callback_sched(executor)
+# traget: executor 
+cbg = app.get_callback_group('cbg_name') # get callback group object
+callback_sched(cbg)
+```
+![Callback_Scheduling_Visualization_sample](../imgs/callback_sched_sample.png)
+- Callback Scheduling Visualization
```

Comment:
Could you fix line breaks?
see: https://tier4.github.io/CARET_doc/pr-54/tips/gallery/#callback-scheduling-visualization